pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
+requires = [
+    "setuptools>=61.2",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "Mathics_Scanner"
+version = "1.3.0"
```

Comment:
How can we this read in from `mathics_scanner/version.py?

Reply:
Thanks. Fixed in 104c114